test/test/jfr/JfrTests.java

```diff
@@ -170,6 +170,26 @@ public void allModeEventOverride(TestProcess p) throws Exception {
         assert events.contains("profiler.Free"); // nativemem profiling
     }
 
+    // Simple smoke test, nothing in particular is tested
```

Comment:
> nothing in particular is tested

This is not a good argument for adding a test.

For example, can we test `--jfropts 9999`, `0`, `-1` etc. magic values?

Reply:
Ideally I'd like to test that setting `jfropts` has "some" effect, but it does not seem to be possible. If an event is disabled for JFR, async-profiler will provide it, and viceversa, so there's no observable effect I can use to write the test.

> For example, can we test --jfropts 9999, 0, -1 etc. magic values?

We don't really do this kind of tests in Async-Profiler, do we?

Reply:
> We don't really do this kind of tests in general, do we?

We don't.
It's disappointing to see debates in the comments on how to unit-test `a + b` or how to name a constant for zero. I exaggerate, but it's roughly how it looks from the outside.
Please guys focus on logical errors and design flaws in your reviews; leave comments about coding style to me :)